libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -88,7 +88,7 @@ func buildDockerMetadataProcessor(cfg *common.Config, watcherConstructor docker.
 			"field":     "source",
 			"separator": string(os.PathSeparator),
 			"index":     config.SourceIndex,
-			"target":    "docker.container.id",
+			"target":    "container.id",
```

Comment:
Was not sure why here the constant was not used. @exekias ok to change this one or could this have other side effects?

Reply:
uhm, probably they were introduced at different points in time, I don't see a reason for it to not use the constant

Reply:
ok, replaced it by the constant then.